x/leverage/keeper/borrows.go

```diff
@@ -210,3 +210,12 @@ func (k Keeper) SetBorrowAPY(ctx sdk.Context, denom string, borrowAPY sdk.Dec) e
 	store.Set(key, bz)
 	return nil
 }
+
+// GetAvailableToBorrow gets the amount available to borrow of a given token.
+func (k Keeper) GetAvailableToBorrow(ctx sdk.Context, denom string) sdk.Int {
+	// Available for borrow = Module Balance - Reserve Amount
+	moduleBalance := k.ModuleBalance(ctx, denom)
+	reserveAmount := k.GetReserveAmount(ctx, denom)
+
```

Comment:
@toteki do we need any zero amount defense checks here? What if `moduleBalance < reserveAmount`?

Reply:
Added in batch